src/scala/scripts/TwitterScroogeGenerator.scala

```diff
@@ -69,11 +70,18 @@ class ScroogeGenerator extends Processor {
     // immediateThriftSrcJars.
     val remoteSelfThriftSources = getIdx(4)
 
+    // Further configuration options for scrooge.
+    val additionalFlags = getIdx(5)
+
     val tmp = Paths.get(Option(System.getenv("TMPDIR")).getOrElse("/tmp"))
     val scroogeOutput = Files.createTempDirectory(tmp, "scrooge")
 
     val scrooge = new Compiler
 
+    if (additionalFlags.contains("--with-finagle")) {
```

Comment:
This should give us an easy spot to add additional flags, without bringing in a full-fledged argument parser.